package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "/usr/local/bin/codeclimate",
+      "description": "The full path to the `codeclimate` binary."
+    },
+    "init": {
+      "type": "boolean",
+      "default": false,
+      "description": "Create a `.codeclimate.yml` file if not found."
+    },
+    "timeout": {
```

Comment:
Change this to a boolean that just disables the timeout entirely, exactly like [this](https://github.com/AtomLinter/linter-flake8/blob/ab20e7c25a74f6d76607de2229834b97880e1aef/package.json#L26).